decorator.rb

```diff
@@ -0,0 +1,13 @@
+require './nameable'
+
+# class decorator
+class Decorator < Nameable
+  def initialize(nameable)
+    @nameable = nameable
+    super()
+  end
+
+  def correct_name
+    @nameable
+  end
```

Comment:
You have done great implementing the correct_name method in the decorator as required. Kindly make sure that it returns the `correct_name` method of the `@nameable `